.ci/pipeline/ci.yml

```
@@ -57,6 +57,65 @@ jobs:
       pip install isort black[jupyter]==24.1.1
       isort --profile black --check . && black --check .
     displayName: 'Linting'
+- job: LinuxCondaRecipe
+  dependsOn: Lint
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Python3.12_Numpy1.26:
+        PYTHON_VERSION: '3.12'
+        NUMPY_VERSION: '1.26'
+      Python3.9_Numpy1.22:
+        PYTHON_VERSION: '3.9'
+        NUMPY_VERSION: '1.22'
+  pool:
+    vmImage: 'ubuntu-22.04'
+  steps:
+  - script: |
+      bash .ci/scripts/install_dpcpp.sh
+    displayName: "dpcpp installation"
+  - script: |
+      conda config --add channels conda-forge
+      conda config --set channel_priority strict
+      conda update -y -q --all
+    displayName: "Conda update"
+  - script: |
+      conda create -y -q -n build-env conda-build conda-verify
+    displayName: "Conda create"
+  - script: |
+      export DPCPPROOT=/opt/intel/oneapi/compiler/latest
+      . /usr/share/miniconda/etc/profile.d/conda.sh
+      conda activate build-env
+      conda build . --python $(PYTHON_VERSION) --numpy $(NUMPY_VERSION)
+    displayName: "Conda build"
```

Since there an option only build without testing I suggest add `test` in the name of the job for better description of the flow.
```suggestion
    displayName: "Conda build and test"
```